BrainPortal/app/models/boutiques_cluster_task.rb

```diff
@@ -76,6 +76,8 @@ def setup #:nodoc:
     descriptor = self.descriptor_for_setup
     self.addlog(descriptor.file_revision_info.format("%f rev. %s %a %d"))
 
+    module_information;
```

Comment:
That's a bad name. With a name like that, it feels it's returning something, but there is no assignment.

What the method does is log informaton about the module. So name it appropriately: "addlog_boutiques_modules_information". All methods that add to the logging system start with "addlog_", after all.

Then, invoke it with self so that it looks like the previous line:

```ruby
   self.addlog_boutiques_module_information
```

For discussion: all these addlogs are added at the time of setup(). I am not sure that's the best place. Maybe in after_final_task_list_saved() ?

Reply:
Finally, if this works out, then remove the addlog() statements in the modules themselves, they become redundant.